infrastructure/matrix/client.go

```diff
@@ -16,4 +16,7 @@ type Client interface {
 	// OnMessage registers a handler that will be called whenever a message is sent to a room
 	// the currently authenticated user is a member of.
 	OnMessage(handler func(roomID room.ID, message message.Message)) error
+
+	// IsCommand returns true if the message passed is an invokation of command
+	IsCommand(message message.Message) bool
```

Comment:
I think this could instead be a method on the `Message` itself.

Reply:
Agree! I will probably do that as a new PR since I don't want to cause conflicts in other PRs.

Reply:
[Committed](https://github.com/Automattic/neurobot/commit/3cc27bf659966e228b7315d1b3042d6f82bf127f) to `command_support_feedback` branch for now